docker-compose.yml

```diff
@@ -32,59 +32,63 @@ services:
             interval: 10s
             retries: 10 
             
-    oe.openelis.org:
-        build:
-          context: .
-        container_name: openelisglobal-webapp 
-        image: openelisglobal-webapp.build     
-        depends_on:
-            - database
-        ports:
-            - "8080:8080"
-            - "8443:8443"
-        restart: always
-        networks:
-          default:
-              ipv4_address: 172.20.1.121
-
-        restart: always
-        environment:
-            - DEFAULT_PW=adminADMIN! 
-            - TZ=Africa/Nairobi
-              # context.xml doesn't seem to be able to pick up environment variables directly, so we are passing them in as CATALINA_OPTS
-            - CATALINA_OPTS= -Ddatasource.url=jdbc:postgresql://database:5432/clinlims -Ddatasource.username=clinlims
-        volumes:
-            -  cert-volume:/etc/openelis-global
-            - ./volume/plugins/:/var/lib/openelis-global/plugins
-            - ./volume/tomcat/oe_server.xml:/usr/local/tomcat/conf/server.xml
-        secrets:
-            - source: datasource.password
-            - source: common.properties
+#    oe.openelis.org:
+#        build:
```

Comment:
you could discard these changes too in this file since they were specific to your local enviroment